vms/platformvm/api/static_service.go

```diff
@@ -98,6 +98,9 @@ type Staker struct {
 	StakeAmount *json.Uint64 `json:"stakeAmount,omitempty"`
 }
 
+// GenesisValidator should to be used for genesis validators only.
+type GenesisValidator = Staker
```

Comment:
```suggestion
type GenesisValidator Staker
```
I think we want to enforce that we provide `GenesisValidator` rather than `Staker` in this PR... So, rather than doing a type alias should we treat them as separate types?

Reply:
Done!